lambo

```diff
@@ -350,7 +355,7 @@ fi
 
 if [[ "$CODEEDITOR" != "" ]]; then
     if isterminaleditor; then
-       $CODEEDITOR .
+       $CODEEDITOR $PROJECTPATH/$PROJECTNAME
```

Comment:
This doens't work; it actually keeps my IDE (Sublime Text) from opening it correctly.